bundles/org.openhab.binding.tado/pom-aggregator.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?><project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
```

Comment:
Is this file still needed? IMO we should try to integrate it in the main pom (in another PR)

Reply:
Yes it’s ok to delete it. Shall l do it or will you?

Reply:
would be great if you could do that

Reply:
Done